docs/Appendix/postgres.md

```diff
@@ -14,9 +14,9 @@ if [ -z "${OS_ID##*debian*}" ]; then
   #Debian/Ubuntu
   wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
   RELEASE=$(lsb_release -cs)
-  echo "deb http://apt.postgresql.org/pub/repos/apt/ ${RELEASE}"-pgdg main | sudo tee  /etc/apt/sources.list.d/pgdg.list
+  echo "deb [arch=amd64] http://apt.postgresql.org/pub/repos/apt/ ${RELEASE}"-pgdg main | sudo tee  /etc/apt/sources.list.d/pgdg.list
   sudo apt-get update
-  sudo apt-get -y install postgresql-13 postgresql-server-dev-13 postgresql-contrib libghc-hdbc-postgresql-dev
+  sudo apt-get -y install postgresql-14 postgresql-server-dev-14 postgresql-contrib libghc-hdbc-postgresql-dev
```

Comment:
If I remember correctly, doing this will still end up installing other versions - and set 13 as default. Something to test/verify

Reply:
Ok, let's test this. I would assume its installing only 14 on a new system but if you for example already have 13 installed and just run apt to upgrade, it will install 14 in addition to 13 but keep 13 at port 5432. But could be wrong.

Reply:
Not critical to run 14 over 13 though even if this is the case.

Reply:
Yip , and it's not different than existing behaviour too 👍🏻